src/depot_gui.cpp

```diff
@@ -974,6 +974,15 @@ struct DepotWindow : Window {
 		}
 	}
 
+	bool last_state;
+	void OnMouseLoop() override
+	{
+		if (last_state != (_shift_pressed & _ctrl_pressed)) {
```

Comment:
These shouldn't be bitwise operators, right?

```suggestion
		if (last_state != (_shift_pressed && _ctrl_pressed)) {
```